requirements.txt

```diff
@@ -1,3 +1,3 @@
 pypif==2.0.0
-citrination_client==4.1.0
-toolz==0.8.2
\ No newline at end of file
+citrination_client==6.3.0
```

Comment:
What about 5.x?  Could you either test that locally or scroll through them in commits in this PR?

Reply:
Yeah I can do that via commits